source/voting.rst

```diff
@@ -0,0 +1,96 @@
+.. _voting:
+
+############
+Voting Items
+############
+
+Decisions about the future of the project are made through discussion with all
+members of the community. All non-sensitive project management discussion takes
+place on the project developers’ mailing list, the issue tracker, and in the
+project meetings mentioned above. Occasionally, sensitive discussion occurs on
+a private list, but the project defaults to open meetings.
+
+The yt-project uses a “consensus seeking” process for making large,
+project-changing decisions. The
+group tries to find a resolution that has no open objections among project
+members. At any point during the discussion, any project member can call for
+a vote, which will conclude two weeks from the call for the vote. Any vote must
+be backed by a YTEP. If no option can gather two thirds of the votes cast, the
+decision is escalated to the Steering Committee, which in turn will use
+consensus seeking with the fallback option of a simple majority vote if no
```

Comment:
It's unclear from this wording when the SC should switch to a majority vote.

Reply:
The process would be 
- first try to find a community consensus 
- if no community consensus is found, then voting within the community would happen via a mailing list vote and/or a YTEP vote
- if no 2/3 majority is found in either of those options, then the steering committee would try to find a consensus (with a month time in case members of the steering committee are unable to participate in the discussion for whatever reason) 
- if the steering committee can't find a consensus, then it falls back to a majority vote.

Reply:
I'm trying to think of a better way to phrase it; do you have any suggestions?

Reply:
I think adding the concrete text you have in this comment will resolve Britton's question.

Reply:
Done!